powerauth-nextstep/src/main/resources/application.properties

```diff
@@ -70,5 +70,15 @@ banner.application.version=@project.version@
 
 logging.config=${POWERAUTH_NEXTSTEP_LOGGING:}
 
+# OpenID Connect (OIDC) Settings
+#powerauth.nextstep.security.auth.type=OIDC
+#spring.security.oauth2.client.registration.nextstep-oidc-client.provider=nextstep-oidc-provider
```

Comment:
Autoconfiguration has to be improved not to fail when the property is specified as empty and the type is not OIDC. In that case, the application should normally start.